pandas/util/_validators.py

```diff
@@ -205,9 +205,39 @@ def validate_args_and_kwargs(fname, args, kwargs, max_fname_arg_count, compat_ar
     validate_kwargs(fname, kwargs, compat_args)
 
 
-def validate_bool_kwarg(value, arg_name):
-    """ Ensures that argument passed in arg_name is of type bool. """
-    if not (is_bool(value) or value is None):
+def validate_bool_kwarg(value, arg_name, none_allowed=True, int_allowed=False):
```

Comment:
if `validate_bool_kwarg` allowed ints through before, wouldn't `int_allowed` need to default to True for backwards compatibility?

Reply:
I checked ``is_bool``, and by default it considers ``int`` as a non-boolean.

```
In [1]: from pandas.core.dtypes.common import is_bool

In [2]: is_bool(1)
Out[2]: False
```